pom.xml

```diff
@@ -37,7 +37,6 @@
   </scm>
 
   <properties>
-    <jenkins.version>2.121</jenkins.version>
```

Comment:
Please retain the explicit jenkins version info in the pom.

Reply:
Done